pkg/analysis/passes/gosec/gosec.go

```diff
@@ -0,0 +1,90 @@
+package gosec
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/grafana/plugin-validator/pkg/analysis"
+	"github.com/grafana/plugin-validator/pkg/analysis/passes/sourcecode"
+	"github.com/grafana/plugin-validator/pkg/logme"
+)
+
+var (
+	goSecNotInstalled = &analysis.Rule{Name: "go-sec-not-installed", Severity: analysis.Warning}
+	goSectIssueFound  = &analysis.Rule{Name: "go-sec-issue-found", Severity: analysis.Warning}
+)
+
+// could be low, medium, high (see gosec docs)
+var targetSeverity = "HIGH"
+
+var Analyzer = &analysis.Analyzer{
+	Name:     "go-sec",
+	Requires: []*analysis.Analyzer{sourcecode.Analyzer},
+	Run:      run,
+	Rules:    []*analysis.Rule{goSecNotInstalled},
+}
+
+func run(pass *analysis.Pass) (interface{}, error) {
+	// only run if sourcecode.Analyzer succeeded
+	sourceCodeDir, ok := pass.ResultOf[sourcecode.Analyzer].(string)
+	if !ok {
+		return nil, nil
+	}
+
+	// check if gosec is installed
+	goSecBin, err := exec.LookPath("gosec")
+	if err != nil {
+		if goSecNotInstalled.ReportAll {
+			logme.Debugln("gosec not installed, skipping gosec analysis")
+			pass.ReportResult(pass.AnalyzerName, goSecNotInstalled, "gosec not installed", "Skipping gosec analysis")
```

Comment:
Good catch. Though I'll return nil nil. I don't want to break the validator for the lack of the binary specially in CI where it might not be installed.